rust-toolchain.toml

```diff
@@ -12,5 +12,7 @@ components = [
     "rust-docs",
     # For formatting
     "rustfmt",
+    # For rust-analyzer
+    "rust-src",
 ]
 profile = "minimal"
```

Comment:
Is this still needed?

Reply:
i think no, based on [these docs](https://rust-lang.github.io/rustup/concepts/profiles.html). should we remove `clippy`, `rust-docs`, and `rustfmt` from the `components` too?

Reply:
Makes sense to me.

Reply:
done in 26b45cfb4e85a7c8cb49828f6c750ea531d137d4